refinery/tool_manager/utils.py

```diff
@@ -222,8 +222,10 @@ def create_tool(tool_launch_configuration, user_instance):
 
     display_name = tool_launch_configuration.get("display_name")
     if not display_name:
-        display_name = "{} {} {}".format(tool.name, tool.creation_date,
-                                         tool.get_owner().username)
+        display_name = "{} {} {}".format(
+            tool.name, tool.creation_date.strftime('%m/%d/%Y %H:%M:%S'),
+            tool.get_owner().username
+        )
```

Comment:
Consider just using one format string, and perhaps naming the slots, too:
```
>>> from datetime import datetime
>>> '{tool} {dt:%m/%d/%Y %H:%M:%S} {user}'.format(tool='tool', dt=datetime.now(), user='user')
'tool 09/04/2018 12:59:34 user'
```

... and maybe use the three-letter month name? and maybe some linking words? But maybe we want this to look more like an ID and less like a name?
```
>>> '{tool} started by {user} on {datetime:%b %d, %Y at %H:%M:%S}'.format(tool='tool', datetime=datetime.now(), user='user')
'tool started by user on Sep 04, 2018 at 13:08:25'
```